CRM/Contact/BAO/Group.php

```diff
@@ -1075,7 +1075,7 @@ public static function getGroupsHierarchy(
     if ($parents) {
       // group can have > 1 parent so parents may be comma separated list (eg. '1,2,5'). We just grab and match on 1st parent.
```

Comment:
Please update this comment to reflect what the code does now.  Just remove the "We just grab and match on 1st parent."?